web/src/transmission.js

```diff
@@ -623,15 +621,15 @@ export class Transmission extends EventTarget {
           r.getElement().scrollIntoView();
           event_.preventDefault();
         }
-      } else if (shift_key) {
+      } else if (code === 'ShiftLeft' || code === 'ShiftRight') {
```

Comment:
For example:
```suggestion
      } else if (key === 'Shift') {
```

Reply:
I'm merely suggesting the `key` can have special characters due to international keyboards. It's just unfortunate that key (cap) location between keyboard layouts are different but I don't want that to be the reason we should switch.

I was hoping to place favor in using `code` not because of their exact location on keyboard, those identifiers of individual keys are loud and clear: `Equal` than `=`, `Space` than ` `, etc. and it can resolve keys without modifier key states.

Although you've made a point about same key in discrete locations that using `code` for it seems overkill.

Do you suggest we mass replace `code` with `key` anyway?

Reply:
Yeah I'd say replace `code` with `key` in this PR, because although the values might not be as clear as `code`, we can be sure it will be correct across keyboard layouts.